android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -38,52 +38,51 @@
 
 public class EditorFragment extends BaseMwmFragment implements View.OnClickListener, EditTextDialogFragment.OnTextSaveListener
 {
-  final static String LAST_LOCALIZED_NAME_INDEX = "LastLocalizedNameIndex";
+  final static String LAST_INDEX_OF_NAMES_ARRAY = "LastIndexOfNamesArray";
 
   private TextView mCategory;
-  private View mCardName;
   private View mCardAddress;
   private View mCardMetadata;
-  private EditText mName;
 
-  private RecyclerView mLocalizedNames;
+  private RecyclerView mNamesView;
 
-  private final RecyclerView.AdapterDataObserver mLocalizedNamesObserver = new RecyclerView.AdapterDataObserver()
+  private final RecyclerView.AdapterDataObserver mNamesObserver = new RecyclerView.AdapterDataObserver()
   {
     @Override
     public void onChanged()
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeChanged(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeInserted(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeRemoved(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeMoved(int fromPosition, int toPosition, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
   };
 
-  private MultilanguageAdapter mLocalizedNamesAdapter;
-  private TextView mLocalizedShow;
-  private boolean mIsLocalizedShown;
+  private MultilanguageAdapter mNamesAdapter;
+  private TextView mNamesCaption;
+  private TextView  mAddLanguage;
```

Comment:
In Android we already have formatter build-in in IDE which is set for our code style. Shortcut for auto format is alt+cmd+L.
Settings can be configured in Preferences-Editor-Code style-Java.


> On Jul 20, 2016, at 10:15 PM, ygorshenin <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/EditorFragment.java <https://github.com/mapsme/omim/pull/3788#discussion_r71588626>:
> 
> >      }
> >    };
> >  
> > -  private MultilanguageAdapter mLocalizedNamesAdapter;
> > -  private TextView mLocalizedShow;
> > -  private boolean mIsLocalizedShown;
> > +  private MultilanguageAdapter mNamesAdapter;
> > +  private TextView mNamesCaption;
> > +  private TextView  mAddLanguage;
> BTW, maybe it worth to set up clang-format for java too? (not in this CL, of course)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/3788/files/38d1d8b3dea10716cada840892605d0cc96b3eca#r71588626>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACKTROwJKu0yaIsw-zfzb1WTWtT9jfaJks5qXnO_gaJpZM4JMjsI>.
>